### ISSUE TITLE
Add Documentation for Auto-documenting/Wiki Resource Module Structure

### ISSUE THREAD

**Issue by PlagueHO** [PowerShell/DscResources#279]
There are now several resource modules that have moved over to using @BrianFarnhill 's auto documentation/wiki generation structure. These modules are SharePointDsc, xNetworking, xStorage and xDFS (with xCertificate being moved). The repo structure and files in the modules are slightly different to the standard module and so we should provide documentation on how this structure differs.

This was raised in this issue: https://github.com/PowerShell/DscResources/issues/278

**Comment by johlju** [comment]
Maybe we should hold off on this until we see if issue https://github.com/PowerShell/DscResource.Tests/issues/225 can be a reality or not?

**Comment by PlagueHO** [comment]
Agreed - hopefully we can find a way to standardize. I just keep running out of time :cry:

**Comment by johlju** [comment]
Yes me too... Looking into automating labels between reviews and closing issues. 🙂
